packages/manager/modules/vps/src/dashboard/vps-dashboard.component.js

```diff
@@ -42,7 +42,7 @@ export default {
     vpsMigrationTask: '<',
     vpsMigrationTaskInError: '<',
     vpsUpgradeTask: '<',
-    canDisplayVpsAnnouncementBanner: '<',
+    isVpsMaintenance: '<',
```

Comment:
IMHO:

```suggestion
    isInMaintenance: '<',
```

sounds more correct. Isn't it?

Reply:
i think first one is more explicit.
thanks for suggestion.

Reply:
Not sure that's the correct way to express if there's a pending maintenance for the given virtual private server.

cc @marie-j

Reply:
isVpsMaintenance would imply that what you're describing is a maintenance and whether it is applied to vps whereas in fact you are describe is a vps which might be in maintenance or not therefore `isInMaintenance` is more accurate

Reply:
`isInMaintenance` is not explicit by reading it first times, we have to read more code to understand what is meaning.
Regarding PCI team, the banner is here to inform customer that his VPS is in maintenance and he will probably enconter issues during it.

For me `isVpsMaintenance` is explicit.